flashinfer/fused_moe/core.py

```
@@ -177,6 +178,39 @@ class GatedActType(IntEnum):
     GeGlu = 1
 
 
+def is_flashinfer_trtllm_moe_supported(
+    dtype_weights: DtypeTrtllmGen,
+    dtype_act: DtypeTrtllmGen,
+    quant_method: Optional[str] = None,
```

Ruff (0.14.3)

Unused function argument: `quant_method`

(ARG001)